demos/common/python/openvino/model_zoo/model_api/README.md

```diff
@@ -0,0 +1,136 @@
+# Python* Model API package
+
+Model API package is a set of wrapper classes for particular tasks and model architectures, simplifying data preprocess and postprocess as well as routine procedures (model loading, asynchronous execution, etc...)
+An application feeds model class with input data, then the model returns postprocessed output data in user-friendly format.
+
+## Package structure
+
+The Python* Model API consists of 3 libraries:
```

Comment:
```suggestion
The Model API consists of 3 libraries:
```
Python mentions are redundant in paragraphs texts, can't we leave them just in headers? At this point it's already obvious that it is python package.

Reply:
Removed in paragraphs texts